### ISSUE TITLE
twisted.internet._ssl is unused and can be deleted

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8775]
|[<img alt="alex's avatar" src="https://avatars.githubusercontent.com/u/772?s=50" width="50" height="50">](https://github.com/alex)| @alex reported|
|-|-|
|Trac ID|trac#8775|
|Type|enhancement|
|Created|2016-08-20 12:51:20Z|

LGTM

<details><summary>Searchable metadata</summary>

```
trac-id__8775 8775
type__enhancement enhancement
reporter__alex alex
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1471697480022394 1471697480022394
changetime__1471699165043033 1471699165043033
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="Alex Gaynor's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Alex Gaynor set status to `closed`|
|-|-|

In changeset 75251943f870d291a916d443abc7832243a8a5f7
```
#!CommitTicketReference repository="" revision="75251943f870d291a916d443abc7832243a8a5f7"
Fixes #8775 -- delete unused _ssl.py
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

In changeset 5ad48a821b7a32b2ee5ff12ac6d74b2df8d15230
```
#!CommitTicketReference repository="" revision="5ad48a821b7a32b2ee5ff12ac6d74b2df8d15230"
Merge alex/patch-1: Delete unused _ssl.py

Author: alex
Reviewers: rodrigc, hawkowl
Fixes: #8775
```
